src/manual/css/landing_page.css

```diff
@@ -0,0 +1,103 @@
+/*
+ *  Copyright (c) 2014 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+ */
+a {
+color: #6fa8dc;
```

Comment:
Ooops, I copied this from the samples repo and forgot to fix indendation (it's not indented at all there).